### ISSUE TITLE
@Getter and @Setter configuration

### ISSUE THREAD

**Issue by pydawan** [projectlombok/lombok#1595]
Would it be possible to generate the default getters and setters along with chained and fluent getters and setters?
Like this:

public class Foo {
   private String name;

   public String getName() {
       return name;
   }

   public void setName(String name) {
      this.name = name;
  }

   public String name() {
      return name;
   }

   public Foo name(String name) {
      this.name = name;
     return this;
   }
}

**Comment by victorwss** [comment]
Just give a look at https://projectlombok.org/features/GetterSetter

**Comment by pydawan** [comment]
Hello victorwss!
I need to generate in both forms together.

**Comment by janmaterne** [comment]
Does this relate to https://github.com/rzwitserloot/lombok/issues/2141?
Does the solution/workaround provided there work for you?

**Comment by Saljack** [comment]
Yes it the same. But there is no workaround. `wither` creates a copy.

**Comment by Maaartinus** [comment]
AFAICT this is a pretty exact duplicate of #2141. IIRC, in some other issue, the project owners stated rather clearly that they consider having two setters to be terrible idea and won't support it.
